packages/components/ovh-reket/src/index.js

```diff
@@ -0,0 +1,74 @@
+import { Reket } from '@ovhcloud/reket-core';
+import { AxiosReketClient } from '@ovhcloud/reket-axios-client';
+
+import { DEFAULT_REQUEST_TYPES } from './constants';
+import { redirectToLoginPage, redirectToLogoutPage } from './redirections';
+
+export const ssoAuthHookFn = (error) => {
+  const rejectedError = error;
+
+  if (
+    rejectedError.status === 403 &&
+    (rejectedError.data.message === 'This session is forbidden' ||
+      rejectedError.data.message === 'This session is invalid')
+  ) {
+    // consider a 401 if previous case is matched
+    rejectedError.status = 401;
+  }
+
+  // Redirect on 401
+  if (rejectedError.status === 401 && !rejectedError.config.preventLogout) {
+    redirectToLogoutPage();
+    return Promise.reject(rejectedError);
+  }
+
+  // If CODE 471 AKA Low-order session
+  if (rejectedError.status === 471) {
+    redirectToLoginPage();
+    return Promise.reject(rejectedError);
+  }
+
+  // Reject the response
+  return Promise.reject(rejectedError);
+};
+
+/**
+ * Create an instance of Reket (configured with axios client)
+ * and predefined hooks for sso auth (if configured)
+ *
+ * @param  {Boolean}              enableSsoAuth   Enable sso auth hooks.
+ * @param  {String|Array<Object>} requestTypes    'default' in order to have default request
+ *                                                types configured
+ *                                                An Array of Object (with `type` and
+ *                                                `urlPrefix` params)
+ *
+ * @return {Reket}          A Reket instance with predefined configuration.
+ */
+export const useReket = (enableSsoAuth = false, requestTypes = 'default') => {
```

Comment:
**question**
Should the enableSsoAuth be truthy by default ? Do we expect to not use ssoAuth in most cases in the context of the manager ?

Reply:
Mmmmh. Good question :-)
In fact here it's falsy by default because when we use Reket with angularJS, we have to wrap the `ssoAuth` hook into `$q`. Like here: https://github.com/ovh/manager/blob/2da9e46715b5bc53b779960e7b5e3519e33920f8/packages/components/ng-ovh-payment-method/src/payment-method.service.js#L7-L17